media/js/mozorg/home/home.js

```diff
@@ -66,4 +70,66 @@
 
     // Video card interactions.
     initVideoEvents();
-})();
+
+    /*
+     * Sticky CTA
+     */
+
+
+    var stickyCTA = document.getElementById('download-firefox-sticky-cta');
+    $(stickyCTA).attr('aria-hidden', 'true');
+
+    // add and remove aria-hidden
+    var primaryTop = new Waypoint({
+        element: document.getElementById('download-firefox-primary-cta'),
+        handler: function(direction) {
+            if(direction === 'down') {
+                // becomes percivable as the user scrolls down
+                $(stickyCTA).removeAttr('aria-hidden');
+            } else {
+                // hidden again as they scroll up
+                $(stickyCTA).attr('aria-hidden', 'true');
+            }
+        }
+    });
+
+    // init dismiss button
+    function initDismissStickyCTA() {
+        // add button
+        var $dismissButton = $('<button>').addClass('sticky-dismiss').text('Dismiss download prompt.');
+        var $stickyWrapper = $(stickyCTA).find('.primary-wrapper');
+        $dismissButton.appendTo($stickyWrapper);
+        // listen for click
+        $dismissButton.on('click', function(){
```

Comment:
Could use the `.one` method here, as this element will only ever be clicked once.